src/models/compass.ts

```diff
@@ -0,0 +1,12 @@
+import {SpecQueryModelGroup} from 'compassql/build/src/model';
+
+export interface Compass {
```

Comment:
I was just thinking we might want to store more then just the results here - but also the other functions of compassql that could come from the server.